modules/administration-guide/pages/installing-che-locally.adoc

```diff
@@ -23,6 +23,6 @@ This section contains instructions for deploying and running {prod} locally, on
 
 Choose one of the following procedures to start the {prod-short} Server using the `{prod-cli}` tool.
 
-* xref:installing-che-on-codeready-containers.adoc[]
+* xref:installing-che-on-red-hat-openshift-local.adoc[]
```

Comment:
Search for all :
`xref:installing-che-on-codeready-containers.adoc[]`
And replace them by:
`xref:installing-che-on-red-hat-openshift-local.adoc[]